ansible/roles/base/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+
+---
+
+    - name: Update apt cache
+      apt: update-cache=yes cache_valid_time=86400
+      become: yes
+      become_method: sudo
```

Comment:
That is true when using vagrant.yml, but if there were another playbook using this role other than vagrant, you can’t guarantee that… I’m not sure which approach would actually be more appropriate there… which is part of the reason I left it in both. It’s probably more correct to remove it from the vagrant.yml file...